src/constants.tsx

```diff
@@ -1,27 +1,41 @@
+import {assertNever} from "./utils";
+
 export const devnetUrl =
   process.env.APTOS_DEVNET_URL || "https://fullnode.devnet.aptoslabs.com/";
 
-export const networks: Record<string, string> = {
+export const networks = {
   local: "http://localhost:8080",
   devnet: devnetUrl,
   test: "https://rosetta.aptosdev.com/",
-  //governance: "https://ait2.aptosdev.com/", // TODO - use the proposals url
 };
 
-export const walletNetworkMap: Record<string, string> = {
-  Devnet: "devnet",
-  Localhost: "local",
-  Testnet: "test",
+export const walleExplorertNetworkMap = (
+  walletNetwork: WalletNetworks,
+): ExplorerNetworks => {
+  switch (walletNetwork) {
+    case "Devnet":
+      return "devnet";
+    case "Localhost":
+      return "local";
+    case "Testnet":
+      return "test";
+    default:
+      return assertNever(walletNetwork);
+  }
 };
 
+export type ExplorerNetworks = keyof typeof networks;
+
+export type WalletNetworks = "Devnet" | "Localhost" | "Testnet";
+
 // Remove trailing slashes
-for (let key of Object.keys(networks)) {
-  if (networks[key].endsWith("/")) {
-    networks[key] = networks[key].slice(0, -1);
+for (let value of Object.values(networks)) {
+  if (value.endsWith("/")) {
+    value = value.slice(0, -1);
```

Comment:
This looks like a bug, it's just creating new strings (not modifying the ones in the networks object).